docs/docs/advanced/debugging.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 10
+title: 🐛 Debugging
+---
+
+In some cases, you'll want to debug your Dart Frog app, and you might have noticed that by adding just the breakpoint, the debugger doesn't stop there like it does by default in a Flutter app.
+Below, there are two quick and easy options for debugging code in Dart Frog.
+
+## Debugging with Dart Frog IDE Extension 🐸
+
+:::caution
+This extension is only available for Visual Studio Code, if you are using another IDE, please refer to the **Debugging by Attaching to Dart Process** section.
+:::
+
+:::info
+If you are interested in an extension for Android Studio or OpenVSX there are some open issues you can follow to learn more about it.
+
+- **Android Studio** [#1326](https://github.com/VeryGoodOpenSource/dart_frog/issues/1326)
+
+- **OpenVSX** [#907](https://github.com/VeryGoodOpenSource/dart_frog/issues/907)
+
+:::
+
+- Go to the Visual Studio Code Marketplace in [here](https://marketplace.visualstudio.com/items?itemName=VeryGoodVentures.dart-frog) and install the extension.
```

Comment:
I think it reads better when we enumerate the steps, like in our "Deploy" sections.

```suggestion
1. Go to the Visual Studio Code Marketplace in [here](https://marketplace.visualstudio.com/items?itemName=VeryGoodVentures.dart-frog) and install the extension.
```

Could you apply the numbers to the following steps too? And also for those in "Debugging by Attaching to Dart Process"?

Reply:
I can add the enumeration list, but just to let you know if you have an enumerated list and in the middle of the list you have for example a note or a tip, the IDE complains because the numbers are not following each other.

![CleanShot 2025-01-28 at 10 08 32@2x](https://github.com/user-attachments/assets/757a552f-8cd9-433d-a46f-273283ced9c4)